### PR TITLE
Add localization

### LINKED ISSUE #669: Add Localization Support to Reports
Discussed at pre-WolfCon 2023 - We should support creating reports in a variety of languages.

This will involve localizing the templates, and any variables used that are not already localized by implementation teams within the FOLIO instance itself.

### PR DESCRIPTION
**Pull request by ClintonWB-EBSCO**
* [x] Resolves #669 
* [x] Add the python-i18n package, with support for JSON configs similar to those used by EBSCO localization.
* [x] Internationalize basic reporting.
* [x] Replace the "Blurbs" concept with standard internationalization.

### CODE REVIEW

**Comment by bltravis** [comment]
@ClintonWB-EBSCO Anything we can do to get the test build/test passing?

**Comment by ClintonWB-EBSCO** [comment]
This touches a lot of files, so it fails the code coverage requirement. 

I/we could try expanding the tests to cover the files and strings, but I'd welcome a review/feedback of it in place.

File: README.md [hunk 1]
```diff
@@ -67,7 +67,66 @@ In order to perform migrations according to this process, you need the following
...
+Extract template files with the `extract_translations` script:
+
+```bash
+python scripts/extract_translations.py
+```
+
+## Internationalizations
+
+Other langauges translations live in `translations/[locale].json`.
+For example, spanish would be `es.json`.
```
Comment (bltravis) [inline README.md:113]: Need to capitalize "spanish"

**Comment by bltravis** [comment]
@kevinpowell1, @banerjek, @marnold-ebsco Would any of you be willing to give this a go on an actual transformation, just to verify the output? I won't have anything ready to go for a day or two.

File: README.md (hunk 1 above)
Comment (ClintonWB-EBSCO) [reply README.md]: Capitalized Spanish and English

**Comment by bltravis** [comment]
I think we have confirmation that the existing functionality for English is preserved, so I'm going to go ahead and merge this.

**Pull request merged**